relenv/runtime.py

```diff
@@ -168,7 +167,8 @@ def bootstrap():
             str(crossroot / "lib" / "python3.10"),
             str(crossroot / "lib" / "python3.10" / "lib-dynload"),
             str(crossroot / "lib" / "python3.10" / "site-packages"),
-        ] + [x for x in sys.path if x.find("site-packages") == -1]
+        ] + [_ for _ in sys.path if _.find("site-packages") == -1]
```

Comment:
Why do you use `.find` as opposed to `"the-string" in variable`?
Performance?

The latter is more readable....

Reply:
Yeah, that's a bad habit of mine. Fixed.